### ISSUE TITLE
not found: mmlspark_2.12-1.0.0-rc3.pom

### ISSUE THREAD

**Issue by yinrong** [Azure/mmlspark#1119]
follow README.md and add dependency to build.sbt, it says dependency not found

![image](https://user-images.githubusercontent.com/4221569/125435315-b40de306-d58c-4a03-a82a-f9abc18001a9.png)

url in the image:
https://mmlspark.azureedge.net/maven/com/microsoft/ml/spark/mmlspark_2.12/1.0.0-rc3/mmlspark_2.12-1.0.0-rc3.pom

**Comment by yinrong** [comment]
I can not list files by https://mmlspark.azureedge.net/maven/com/microsoft/ml/spark/

Latest build on GIT homepage is not found.

However,I found a version in closed issue which downloads successfully:
1.0.0-rc3-59-bf337941-SNAPSHOT

**Comment by yinrong** [comment]
@imatiach-msft you may close this issue, if this is the right way.

**Comment by imatiach-msft** [comment]
@yinrong rc3 is on scala 2.11, not 2.12.  This url actually works for me:
https://mmlspark.azureedge.net/maven/com/microsoft/ml/spark/mmlspark_2.11/1.0.0-rc3/mmlspark_2.11-1.0.0-rc3.pom

Note spark 3.0+ and scala 2.12 support is on latest master now (and for all builds on master), but rc3 release was still on older scala/spark versions.

**Comment by pfung** [comment]
@imatiach-msft Hello, I want to get the latest build with your new feature #1066 but I don't know where the snapshot jar is. I've tried to download from https://mmlspark.azureedge.net/maven/com/microsoft/ml/spark/mmlspark_2.12/1.0.0-rc3-147-84d8d246-SNAPSHOT/mmlspark_2.12-1.0.0-rc3-147-84d8d246-SNAPSHOT.jar but it doesn't exist.

**Comment by imatiach-msft** [comment]
@pfung  I hope this issue might help to download the jar:
https://github.com/Azure/mmlspark/issues/818

```
mvn dependency:get -DremoteRepositories="https://mmlspark.azureedge.net/maven" -Dartifact="com.microsoft.ml.spark:mmlspark_2.11:1.0.0-rc1" -Ddest=.
```

**Comment by imatiach-msft** [comment]
@pfung this worked for me:

mvn dependency:get -DremoteRepositories="https://mmlspark.azureedge.net/maven" -Dartifact="com.microsoft.ml.spark:mmlspark:1.0.0-rc3-147-84d8d246-SNAPSHOT" -Ddest=.

it looks like the _2.<11 or 12> was dropped recently

**Comment by pfung** [comment]
> @pfung this worked for me:
> 
> mvn dependency:get -DremoteRepositories="https://mmlspark.azureedge.net/maven" -Dartifact="com.microsoft.ml.spark:mmlspark:1.0.0-rc3-147-84d8d246-SNAPSHOT" -Ddest=.
> 
> it looks like the _2.<11 or 12> was dropped recently

@imatiach-msft Thank you very much! I can download the jars now.

**Comment by mhamilton723** [comment]
Hey @pfung and @yinrong thanks for the patience. After our refactor splitting the library into sub-components recently we had to drop the _2.12 prefix. 

Also since 1.0.0-rc3 we bumped to 2.12 from 2.11 so the maven artifacts before the refactor but after the version bump will have the 2.12 postfix. 


The latest master coordinates are:

MMLSpark Build and Release Information
Maven Coordinates
com.microsoft.ml.spark:mmlspark:1.0.0-rc3-148-87ec5f74-SNAPSHOT

Maven Resolver
https://mmlspark.azureedge.net/maven

Documentation Pages:
Scala Documentation: https://mmlspark.blob.core.windows.net/docs/1.0.0-rc3-148-87ec5f74-SNAPSHOT/scala/index.html
Python Documentation : https://mmlspark.blob.core.windows.net/docs/1.0.0-rc3-148-87ec5f74-SNAPSHOT/pyspark/index.html
